hammr/utils/hammr_utils.py

```diff
@@ -240,3 +240,31 @@ def create_user_ssh_key(api, login, sshKey):
         printer.out("Impossible to create sshKey ["+mySshKey.name+"]", printer.ERROR)
         return 2
     return key
+
+def is_uri_based_on_appliance(uri):
+    match = re.match( r'users/.+/appliances/*', uri)
+    if match:
+        return True
+    else:
+        return False
+
+def is_uri_based_on_scan(uri):
+    match = re.match( r'users/.+/scannedinstances/[0-9]+/scans/*', uri)
+    if match:
+        return True
+    else:
+        return False
+
+def extract_scannedinstance_id(image_uri):
+    match = re.match( r'users/.+/scannedinstances/([0-9]+)/.+', image_uri)
```

Comment:
can be  users/[^/]+/scannedinstances/([0-9]+)($|/)
Current does not work for "users/guest/scannedinstances/12" or "users/guest/scannedinstances/12/"